package.json

```diff
@@ -104,17 +99,21 @@
     "eslint": "^7.5.0",
     "eslint-config-prettier": "^6.11.0",
     "eslint-plugin-import": "^2.22.0",
+    "eslint-import-resolver-alias": "^1.1.2",
+    "eslint-plugin-prettier": "^3.1.4",
```

Comment:
This dependency was missing which was one of the causes for prettier not formatting correctly on save in vscode. It also was preventing lint errors from surfacing via yellow squiggles. There were errors in the terminal output from Eslint which was halting it.